packages/shared/src/components/post/common/PostClickbaitShield.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { type ReactElement } from 'react';
+import classNames from 'classnames';
+import { Button, ButtonSize, ButtonVariant } from '../../buttons/Button';
+import { ShieldCheckIcon, ShieldIcon, ShieldWarningIcon } from '../../icons';
+import {
+  useActions,
+  usePlusSubscription,
+  useViewSize,
+  ViewSize,
+} from '../../../hooks';
+import { SimpleTooltip } from '../../tooltips';
+import { useLazyModal } from '../../../hooks/useLazyModal';
+import { ActionType } from '../../../graphql/actions';
+import { useSettingsContext } from '../../../contexts/SettingsContext';
+import { LazyModal } from '../../modals/common/types';
+import { FilterMenuTitle } from '../../filters/helpers';
+
+import { ClickbaitTrial } from '../../plus/ClickbaitTrial';
+import { useSmartTitle } from '../../../hooks/post/useSmartTitle';
+import type { Post } from '../../../graphql/posts';
+
+export const PostClickbaitShield = ({ post }: { post: Post }): ReactElement => {
+  const { openModal } = useLazyModal();
+  const { isPlus, showPlusSubscription } = usePlusSubscription();
+  const { checkHasCompleted } = useActions();
+  const { flags } = useSettingsContext();
+  const { clickbaitShieldEnabled } = flags;
+  const { fetchSmartTitle, usedTrial } = useSmartTitle(post);
+  const isMobile = useViewSize(ViewSize.MobileL);
+
+  if (!showPlusSubscription) {
+    return null;
+  }
+
+  if (!isPlus) {
+    const hasUsedFreeTrial = checkHasCompleted(ActionType.FetchedSmartTitle);
+    return (
+      <div
+        className={classNames(
+          'mt-6 flex flex-wrap items-center text-text-tertiary typo-callout tablet:mt-1',
+          !usedTrial &&
+            'rounded-12 border border-border-subtlest-tertiary px-3 py-2',
+        )}
+      >
+        <Button
+          className="relative mr-2 font-normal"
+          size={ButtonSize.XSmall}
+          icon={
+            usedTrial ? (
+              <ShieldCheckIcon className="text-status-success" />
+            ) : (
+              <ShieldWarningIcon className="text-accent-cheese-default" />
+            )
+          }
+          iconSecondaryOnHover
+        />
+
+        {usedTrial ? (
+          <>
+            This title was optimized with Clickbait Shield
+            <ClickbaitTrial />
+          </>
+        ) : (
+          <>
+            This title could be clearer and more informative.
+            <Button
```

Comment:
You probably want to render as tag=a here?
Think also underline has some better use-case somewhere else already

Reply:
It's not a link though, it triggers a modal

Reply:
Just semantic right?
You can still hijack the onclick function in that case.

Reply:
Changed to `tag=a`, and remove bg on hover